src/core/search/searchapi.js

```diff
@@ -129,6 +129,42 @@ export default class SearchApi {
       .then(response => response.json());
   }
 
+  /**
+   * Initiate a Julian Search
+   * @param {boolean} includeLastName
+   * @param {boolean} useUserName
+   * @param {string} additionalSearchText
+   */
+  julianSearch (includeLastName, useUserName, additionalSearchText) {
+    let name = includeLastName ? 'Julian Brown' : 'Julian';
+    name = useUserName ? 'jbrown' : name;
+
+    if (additionalSearchText && typeof additionalSearchText !== 'string') {
+      throw new AnswersCoreError('additionalSearchText must be a string', 'julianSearch');
+    }
+
+    const query = `${name} ${additionalSearchText}`;
+
+    let request = new ApiRequest({
+      endpoint: '/v2/accounts/me/answers/query',
+      apiKey: this._apiKey,
+      version: this._version,
+      params: {
+        'input': query,
+        'experienceKey': this._experienceKey,
+        params: {
```

Comment:
ah sorry just caught this-- any reason this is duplicated?

Reply:
must have been my autocomplete, I can fix